### PR TITLE
[Feature/CON-126]: add ref and onClick func for tracking

### PR DESCRIPTION
**Pull request by hassanraza0054**
Ticket: https://aesoponline.atlassian.net/browse/CON-126

Related web-ui ticket: https://github.com/aesop/aesop-web-ui/compare/consideration...feature/CON-126?expand=1

### CODE REVIEW

File: src/components/FullWidthHeroScroll/FullWidthHeroScroll.js [hunk 1]
```diff
@@ -10,110 +10,118 @@ import { BackgroundElement } from './components/BackgroundElement/index.ts';
...
+  },
+);
 
 FullWidthHeroScroll.propTypes = {
   backgroundImage: PropTypes.object,
   backgroundMediaType: PropTypes.oneOf(['image', 'video']),
   backgroundVideo: PropTypes.object,
   copy: PropTypes.node,
   cta: PropTypes.shape({
+    englishLabel: PropTypes.string,
```
Comment (MkMan) [inline src/components/FullWidthHeroScroll/FullWidthHeroScroll.js:123]: Where does this get used?

**Review by MkMan — COMMENTED** [review]
Why are there changes to `package-lock.json`?

File: src/components/FullWidthHeroScroll/FullWidthHeroScroll.js (hunk 1 above)
Comment (hassanraza0054) [reply src/components/FullWidthHeroScroll/FullWidthHeroScroll.js:123]: Ah this gets used as part of the CTA tracking callback.

**Comment by hassanraza0054** [comment]
> Why are there changes to `package-lock.json`?

Not sure why my package-lock got changed. Let me have a look

File: src/components/FullWidthHeroScroll/FullWidthHeroScroll.js (hunk 1 above)
Comment (MkMan) [reply src/components/FullWidthHeroScroll/FullWidthHeroScroll.js:123]: Ok. I couldn't references to that variable here. I am guessing it gets passed in as part of something like `...cta` in Web UI?

**Review by MkMan — APPROVED** [review]

**Review by zack76 — APPROVED** [review]

**Pull request merged**